<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextWindow</class>
 <widget class="QWidget" name="TextWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1411</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tram17</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton 
{ 
	font: 75 14pt &quot;DS Eraser Cyr&quot;;
background-color: rgb(212, 212, 212);
border-style: outset;
border-radius: 15px;
border-width: 3px;
border-color: white;
padding: 7px;
}
QPushButton:hover
{ 
background-color: black;
color: white;
 }
QPushButton:pressed
{
border-style: inset;
}
QWidget#TextWindow {
background-color: rgb(212, 212, 212);}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="infoFrame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="infoLayout">
      <property name="leftMargin">
       <number>11</number>
      </property>
      <property name="rightMargin">
       <number>11</number>
      </property>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="timeWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>DS Eraser Cyr</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Время игры: </string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="timeLabel">
           <property name="font">
            <font>
             <family>DS Eraser Cyr</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="soundButton">
        <property name="text">
         <string>Музыка: ВКЛ</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="saveLabel">
        <property name="font">
         <font>
          <family>DS Eraser Cyr</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Сохранено...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitButton">
        <property name="text">
         <string>Выход в меню</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>timeWidget</zorder>
     <zorder>quitButton</zorder>
     <zorder>saveButton</zorder>
     <zorder>saveLabel</zorder>
     <zorder>soundButton</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="textWidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <family>Verdana</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(212, 212, 212);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="autoFormatting">
         <set>QTextEdit::AutoNone</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="decisionsWidget" native="true">
        <layout class="QVBoxLayout" name="decisionsLayout">
         <property name="leftMargin">
          <number>9</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QPushButton" name="nextButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Далее</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
